documentation/modules/exploit/linux/local/cve_2021_4034_pwnkit_lpe_pkexec.md

```diff
@@ -0,0 +1,319 @@
+## Vulnerable Application
+
+The `check` method uses the exploit without a payload.  The patched version
+will exit without any arguments and give the help menu.  Unpatched versions
+give the error that the shell was not in the shells folder.
+
+Last Vulnerable Ubuntu packages:
+- 20.04: 0.105-26ubuntu1.2
+- 21.10: 0.105-31ubuntu0.1
+- 18.04: 0.105-20ubuntu0.18.04.6
+
+Source: https://github.com/cyberark/PwnKit-Hunter/blob/main/CVE-2021-4034_Finder.py
+
+Last Vulnerable Debian Packages:
+- stretch: 0.105-18+deb9u2
+- buster: 0.105-25+deb10u1
+- bullseye: 0.105-31+deb11u1
+
+Source: https://github.com/cyberark/PwnKit-Hunter/blob/main/CVE-2021-4034_Finder.py
+
+Vulnerable ContOS Packages:
+- polkit-0.112-5.ael7b
+- polkit-0.112-13.p1.el7a
+- polkit-0.96-2.el6
+- polkit-0.96-2.el6_0.1
+- polkit-0.96-5.el6_4
+- polkit-0.96-7.el6
+- polkit-0.96-7.el6_6.1
+- polkit-0.96-11.el6
+- polkit-0.96-11.el6_10.1
+- polkit-0.112-1.el7
+- polkit-0.112-5.el7
+- polkit-0.112-6.el7_2
+- polkit-0.112-7.el7_2.2
+- polkit-0.112-7.el7_2.3
+- polkit-0.112-7.el7_2
+- polkit-0.112-9.el7
+- polkit-0.112-11.el7_3
+- polkit-0.112-12.el7_3
+- polkit-0.112-12.el7_4.1
+- polkit-0.112-14.el7
+- polkit-0.112-14.el7_5.1
+- polkit-0.112-17.el7
+- polkit-0.112-18.el7
+- polkit-0.112-18.el7_6.1
+- polkit-0.112-18.el7_6.2
+- polkit-0.112-22.el7
+- polkit-0.112-22.el7_7.1
+- polkit-0.112-26.el7
+- polkit-0.115-6.el8
+- polkit-0.115-9.el8
+- polkit-0.115-9.el8_1.1
+- polkit-0.115-11.el8
+- polkit-0.115-11.el8_2.1
+- polkit-0.115-11.el8_3.2
+- polkit-0.115-11.el8_4.1
+- polkit-0.115-12.el8
```

Comment:
This should all be fixed to render properly in the markdown from `info -d`.

![image](https://user-images.githubusercontent.com/58950994/156424449-fe80a77c-e365-42a8-910c-8c7d9d2a03b0.png)

Reply:
That's annoying.... Rubymine renders that syntax as a list....
![image](https://user-images.githubusercontent.com/17987018/156440761-e5f76ac3-7e35-490c-8808-fb880ca2054e.png)